src/main.cpp

```diff
@@ -34,12 +34,14 @@
 #include "status/StatusManager.h"
 #include "batterymonitor.h"
 #include "logging/Logger.h"
+#include <arduino-timer.h>
```

Comment:
Personally would include GlobalVars.h rather than repeating all imports already present there, to clarify the link between the two.